cmd/package.go

```diff
@@ -642,7 +642,7 @@ func persistBootCmdsIntoFiles(runYamlData []byte, mpmFolder, customBoot string,
 		}
 
 		// Persist to file.
-		cmdFile := filepath.Join(targetFolder, confName)
+		cmdFile := filepath.Join(targetFolder, prefix+confName)
```

Comment:
My initial idea was to have the run config based on their names, i.e. without any prefix. So, if I have a package `spark`, it would save two configs: `master` and `slave` regardless of whether it is composed as a top-level package or one that is being included into another one. In case the new app package that includes `spark` names one of its configs `master` it would override the original file (due to the order of package processing).

The `prefix` itself would only be used to link to default run configs. For example, if `spark` marked `slave` to be a default, there would be the following files

```
/run/master
/run/spark-default -> /run/slave
/run/slave
```

To me, it is a bit strange that run config is named differently in case when we are composing the package itself compared to when we are composing it as an included package.